docs/firmware/device-sdk/authentication/concepts-auth.md

```diff
@@ -0,0 +1,10 @@
+---
+title: 'Concepts - Device Auth'

```

Comment:
These are fine to leave for now but would like them to drop the "Concepts - " and be more descriptive, like "Sending logs."